v2/api/network/customizations/network_security_group_extension_test.go

```diff
@@ -1,70 +0,0 @@
-/*
-Copyright (c) Microsoft Corporation.
-Licensed under the MIT license.
-*/
-
-package customizations
-
-import (
-	"testing"
-
-	. "github.com/onsi/gomega"
-
-	network "github.com/Azure/azure-service-operator/v2/api/network/v1api20201101"
-	"github.com/Azure/azure-service-operator/v2/internal/util/to"
-)
-
-func Test_FuzzySetNetworkSecurityGroupsSecurityRules(t *testing.T) {
```

Comment:
These files (all of the deletions in this folder) shouldn't have been deleted. Can you revive them?

Reply:
fixed